runtimes/cloudformation/cfnpatcher/template.go

```diff
@@ -29,26 +29,42 @@ func GetValueFromTemplate(what *gabs.Container) (string, *gabs.Container) {
 		result = v
 		fallback = nil
 	default:
-		fallback = what
 		result = "placeholder: " + what.String()
+		fallback = what
```

Comment:
Typo to move this line?

Reply:
just for consistency with the case immediately above